### PR TITLE
Reload FBO account

### PR DESCRIPTION
**Pull request by kenanmamedoff**

### CODE REVIEW

File: brd-ios/breadwallet/Helpers/UserManager.swift [hunk 1]
```diff
@@ -91,13 +91,19 @@ class UserManager: NSObject {
             case .success(let usdAccount):
                 guard let usdAccount else { return }
                 
+                if self?.usdAccount?.status != usdAccount.status {
+                    Store.trigger(name: .reloadUsaAccount)
```
Comment (dgacevic) [inline brd-ios/breadwallet/Helpers/UserManager.swift:95]: usa? think it's a typo, also didn't you say you want to use fbo? :)

File: brd-ios/breadwallet/src/ViewControllers/HomeScreenViewController.swift [hunk 2]
```diff
@@ -128,14 +128,17 @@ class HomeScreenViewController: UIViewController, UITabBarDelegate, Subscriber {
...
-                self?.totalCryptoAssetsAmountLabel.alpha = 1
-                self?.usdAccountViewController.view.alpha = 0
-                self?.assetListTableViewController.view.alpha = 1
+                self?.setAssetListVisible(segment: segment)
                 
             case .usd:
+                guard UserManager.shared.profile != nil else {
+                    view.setup(with: viewModel)
+                    
+                    self?.didTapCreateAccountFromPrompt?()
```
Comment (dgacevic) [inline brd-ios/breadwallet/src/ViewControllers/HomeScreenViewController.swift:137]: Make the callback name specific to fbo

Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/ViewControllers/HomeScreenViewController.swift:137]: it is not a fbo related callback.

File: brd-ios/breadwallet/Helpers/UserManager.swift (hunk 1 above)
Comment (kenanmamedoff) [reply brd-ios/breadwallet/Helpers/UserManager.swift:95]: i said i want to but in a refactor. not now.

File: brd-ios/breadwallet/src/ViewControllers/HomeScreenViewController.swift (hunk 2 above)
Comment (dgacevic) [reply brd-ios/breadwallet/src/ViewControllers/HomeScreenViewController.swift:137]: Got it.

File: brd-ios/breadwallet/Helpers/UserManager.swift (hunk 1 above)
Comment (dgacevic) [reply brd-ios/breadwallet/Helpers/UserManager.swift:95]: Okay, but it was probably meant to be `reloadUsdAccount`, no?

Comment (kenanmamedoff) [reply brd-ios/breadwallet/Helpers/UserManager.swift:95]: no

**Review by dgacevic — APPROVED** [review]

**Pull request merged**